executor/explainfor_test.go

```diff
@@ -185,3 +185,21 @@ func (s *testPrepareSerialSuite) TestExplainForConnPlanCache(c *C) {
 		"  └─TableFullScan_5 10000.00 cop[tikv] table:t keep order:false, stats:pseudo",
 	))
 }
+
+func (s *testPrepareSerialSuite) TestExplainDotForExplainPlan(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+
+	rows := tk.MustQuery("select connection_id()").Rows()
+	c.Assert(len(rows), Equals, 1)
+	connID := rows[0][0].(string)
+	tk.MustQuery("explain select 1").Check(testkit.Rows(
+		"Projection_3 1.00 root  1->Column#1",
+		"└─TableDual_4 1.00 root  rows:1",
+	))
+
+	tkProcess := tk.Se.ShowProcess()
+	ps := []*util.ProcessInfo{tkProcess}
+	tk.Se.SetSessionManager(&mockSessionManager1{PS: ps})
+
+	tk.MustQuery(fmt.Sprintf("explain for connection %s", connID)).Check(nil)
```

Comment:
This line should be changed, too

Reply:
Actually, in my original vision, the first one does not need to be added(it is just an explain), and this one needs to be added.